spec.bs

```diff
@@ -819,12 +819,26 @@ To <dfn>fill in a pending fenced frame config</dfn> given a [=fenced frame confi
 1. Set |pendingConfig|'s [=fenced frame config/mapped url=]'s [=mapped url/value=] to
    |replacedAdURL|.
 1. Let |adSize| be |winningBid|'s [=generated bid/ad descriptor=]'s [=ad descriptor/size=].
+1. Let |concreteAdSize| be the result of TODO: resolving screen-relative sizes to pixel values for
+   |adSize|.
+1. Let |adWidthString| and |adHeightString| be the result of TODO: converting |concreteAdSize| to
+   strings.
+   Issue: Resolve this TODO.

```

Comment:
these "Issues" are not rendered as Issue blocks in the preview. I think you need to add an extra empty line before "Issue:".
Also, change this to `Issue: converting |concreteAdSize| to strings. (<a href....)`, instead of a separate TODO and a "resolve this TODO".
Apply to all issues in this PR.

Reply:
looks like it's working now, thanks